pkg/clustermanager/manager.go

```diff
@@ -117,6 +117,9 @@ func (m *Manager) markUnavailable(clusterName string) {
 }
 
 func (m *Manager) start(ctx context.Context, cluster *v3.Cluster, controllers, clusterOwner bool) (*record, error) {
+	if cluster.DeletionTimestamp != nil {
+		return nil, nil
+	}
```

Comment:
We should remove this and just revert the change in `ToRESTConfig`, that gets us back to the old logic.

Reply:
So remove the `&& cluster.Status.Driver != v32.ClusterDriverRKE` check and the comment

Reply:
The timestamp check is in `ToRESTConfig` is the whole reason the imported cluster cleanup is failing. If I put it back there then I would have to write a new function to get a rest config for deleting clusters.

Reply:
Yup, never mind